src/mito2/src/cache/write_cache.rs

```diff
@@ -65,11 +70,60 @@ impl WriteCache {
         request: SstUploadRequest,
         write_opts: &WriteOptions,
     ) -> Result<Option<SstInfo>> {
-        // TODO(yingwen): Write to the local store and then upload.
-        // Now we write to the remote and ignore local cache.
-        let mut writer =
-            ParquetWriter::new(request.upload_path, request.metadata, request.remote_store);
-        writer.write_all(request.source, write_opts).await
+        let _timer = WRITE_AND_UPLOAD_ELAPSED_TOTAL.start_timer();
```

Comment:
We can reuse the `FLUSH_ELAPSED` metrics and add two label values:
- write_parquet
- upload


It'd be helpful to collect the time of writing sst and uploading sst separately.